### ISSUE TITLE
Not speak Italian

### ISSUE THREAD

**Issue by Maverick1983** [rsxdalv/tts-generation-webui#210]
I created voice from wav file, with italian audio, but when I use this .npz for input text, it's complete different. Sometime spoke english, sometime spoke Francaise...

**Comment by rsxdalv** [comment]
Yes, the default is English only. I haven't seen an Italian tokenizer but
it might exist. If you find one I can easily add it to the repo. The
official project for this "technology module" is git mylo bark voice clone.

**Comment by Maverick1983** [comment]
Ok, I know this repo. I'm working to create Italian model.

Thanks

Il gio 2 nov 2023, 02:11 Roberts Slisans ***@***.***> ha
scritto:

> Yes, the default is English only. I haven't seen an Italian tokenizer but
> it might exist. If you find one I can easily add it to the repo. The
> official project for this "technology module" is git mylo bark voice
> clone.
>
>

**Comment by rsxdalv** [comment]
Great! Once you have a model, I can add it to the webui.
